src/callbacks/AaveV2FlashLoanCallback.sol

```diff
@@ -65,15 +65,15 @@ contract AaveV2FlashLoanCallback is IAaveV2FlashLoanCallback, CallbackFeeBase {
         }
 
         // Approve assets for pulling from Aave Pool
-        address feeCollector = IRouter(router).feeCollector();
+        bytes32 defaultReferral = IRouter(router).defaultReferral();
```

Comment:
Would like to move this line to the below `if (charge) { block` to reduce cold storage read cost in most cases, and as a multiple-assets flash loan is rare.